.github/workflows/release-app.yml

```
@@ -92,8 +92,8 @@ jobs:
         run: |
           docker buildx imagetools create ghcr.io/wordpress/openverse-${{ inputs.app }}:${{ inputs.image-sha }} --tag ghcr.io/wordpress/openverse-${{ inputs.app }}:${{ steps.tag.outputs.image-tag }}
 
-          if [ "${{ inputs.app }}" = "api" ]; then
-            docker buildx imagetools create ghcr.io/wordpress/openverse-api_nginx:${{ inputs.image-sha }} --tag ghcr.io/wordpress/openverse-api_nginx:${{ steps.tag.outputs.image-tag }}
+          if [[ "${{ inputs.app }}" == "api" ]] || [[ "${{ inputs.app }}" == "frontend" ]]; then
```

This condition can be simplified:

```suggestion
          if [[ "frontend api" == *"${{ inputs.app }}"* ]]; then
```

Check out this SO answer: https://stackoverflow.com/questions/229551/how-to-check-if-a-string-contains-a-substring-in-bash

---

This change caused a CI linting error:
```
.github/workflows/release-app.yml:92:9: shellcheck reported issue in this script: SC2193:warning:3:22: The arguments to this comparison can never be equal. Make sure your syntax is correct [shellcheck]
   |
92 |         run: |
``` 
So, I reverted the change.